packages/client/package.json

```diff
@@ -11,5 +11,14 @@
     "client"
   ],
   "license": "MIT",
-  "private": false
+  "private": false,
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.8.0",
+    "@typescript-eslint/parser": "^3.8.0",
+    "eslint": "^7.6.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "prettier": "^2.0.5"
```

Comment:
I added `eslint`, and its configurations about `prettier`, `airbnb`, `typescript`.
I think it's done for making our code clean.